### PR TITLE
Enable http redirect with SSL plug

### PR DESCRIPTION
**Pull request by KaylaBrady**
This PR enables SSL for certain endpoints (only will exclude healthcheck & auth endpoints). Referred to [this Arrow PR ](https://github.com/mbta/arrow/pull/6) for my approach here.

### CODE REVIEW

File: config/prod.exs [hunk 1]
```diff
@@ -2,7 +2,8 @@ use Mix.Config
 
 config :draft, Draft.Repo,
   aws_rds_mod: ExAws.RDS,
-  show_sensitive_data_on_connection_error: false
+  show_sensitive_data_on_connection_error: false,
+  ssl: true
```
Comment (KaylaBrady) [inline config/prod.exs:6]: Unrelated SSL change -- noticed Arrow had SSL enabled for the connection to the Postgres DB, following suit for Draft.

Comment (lemald) [reply config/prod.exs:6]: Good question, I'm not actually sure of the background of how the Ecto connectivity stuff was set up for Arrow. That might actually be an @ianwestcott question in case this involves any messing around with SSL certificates in AWS.

**Review by lemald — APPROVED** [review]
Looks good to me!
